0-ad-history-mod-package/simulation/templates/units/han/champion_infantry_spearman_academy.xml

```diff
@@ -6,6 +6,12 @@
     <SpecificName>Hǔ Bēn</SpecificName>
     <SelectionGroupName>units/han/champion_infantry_spearman</SelectionGroupName>
     <Icon>units/han/champion_infantry_swordsman.png</Icon>
+    <History>
+        The Han empire's capital, Chang'an (206 BC to 9 AD, 190 BC to 195 BC), later Louyang (23–190 AD, 196 AD), and Xuchang (196–220 AD), was protected by various guard corps.
+        The largest and strategically most important was the Northern Army, which was stationed in the surroundings of the capital. It consisted of five regiments: the Archers, the Footsoldiers, the Elite Cavalry, the Garrison Cavalry, and the Chang River Regiment. Latter comprised non-Chinese auxiliary troops. Each regiment contained around 1,000 professionally trained soldiers. There was also a Southern Army of similar size; however, it lasted shorter and was of lower importance.
+        The palace guards patrolled the capital's multiple palaces' gates, towers, and walls. They numbered around 2000 and were managed by the Minister of Guards, one of the nine ministers.The Minister of the Household commanded several guard squads as well. The two militarily most valuable were the "虎賁 huben," Rapid as Tigers (which the Champion Spearman represents), and the "羽林 yulin," Feathered Forest. Both were between 1000 and 2000 men strong and responsible for the emperor's immediate personal security. Memberships in these units were passed on to the next generation.
```

Comment:
The Chinese names are unfortunately kind of distracting.

Reply:
Do you only mean the Chinese words or also the Latinized transcriptions?

Reply:
Everything in the quotations

Reply:
okay